Assets/Scripts/API/BlocksFile.cs

```diff
@@ -584,8 +584,7 @@ private void ReadRmbFldHeader(BinaryReader reader, int block)
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Unknown3 = reader.ReadUInt32();
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Unknown4 = reader.ReadUInt32();
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].FactionId = reader.ReadUInt16();
-                blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Sector = reader.ReadInt16();
```

Comment:
Please do not change this. The LocationID is actually used by quest system - the "sector" value is useless to me.